lib/prepare_release.js

```diff
@@ -382,8 +523,11 @@ export default class ReleasePreparation {
     const arr = data.split('\n');
     const allCommits = this.getChangelog();
     const notableChanges = this.getBranchDiff({ onlyNotableChanges: true });
-    const releaseHeader = `## ${date}, Version ${newVersion}` +
+    let releaseHeader = `## ${date}, Version ${newVersion}` +
                           ` ${releaseInfo}, @${username}\n`;
+    if (isSecurityRelease) {
+      releaseHeader += 'This is a security release.';
```

Comment:
I think it might need an extra line break:

```suggestion
      releaseHeader += '\nThis is a security release.';
```